features/step_definitions/visitor_can_create_new_task_steps.rb

```diff
@@ -0,0 +1,24 @@
+Then("I should be on New Task page") do
+    visit new_task_path
+end
+
+Then("I fill in {string} with {string}") do |field, content|
+    fill_in field, with: content
+end
+
+Then("I click on {string} link") do |new_task|
+    click_link new_task
+end
+
+Then("there should be a {string} in the database") do |task_title|
+    task = Task.find_by(title: task_title)
+    expect(current_path).to eq root_path(task)
```

Comment:
`root_path(task)`?? The `root_path` does not take arguments. You probably mean `task_path(task)`